src/PlayButton.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+import React from 'react';
+import AriaDisablingButton from './AriaDisablingButton';
+import { injectIntl } from 'react-intl';
+import { ReactComponent as PlayIcon } from './svg/Play.svg';
+import './PlayButton.scss';
+
+type PlayButtonProps = {
+    intl: any,
+    interpreterIsRunning: boolean,
+    runButtonDisabled: boolean,
+    onClickRunButton: () => void
+};
+
+class PlayButton extends React.Component<PlayButtonProps, {}> {
+    render() {
+        return (
+            <div className='PlayButton-container'>
```

Comment:
I think the `PlayButton-container` div and styling would be better located in the App because they are dependent on the location within a grid. Which is the responsibility of the App layout.